src/test/java/org/takes/misc/HrefTest.java

```diff
@@ -72,6 +73,22 @@ void buildsUriWithoutParams() {
         );
     }
 
+    /**
+     * Href can get query parameters.
+     */
+    @Test
+    void getQueryParameters() {
+        final String uri = "http://a.example.com?param1=hellow&param2=world";
+        MatcherAssert.assertThat(
+            new Href(uri).param("param1"),
+            new HasValues<>("hellow")
+        );
+        MatcherAssert.assertThat(
+            new Href(uri).param("param2"),
+            new HasValues<>("world")
+        );
+    }
+
```

Comment:
@baudoliver7 It would be nice to have some escaped sequences in parameters as well for tests.

Reply:
@andreoss Do you mean for example escape `&` by `%26` ?

Reply:
@baudoliver7 Yes, and test that it being extracted correctly

Reply:
@andreoss Got it now :)